integration_tests/integration/createLicence.cy.ts

```diff
@@ -102,6 +103,8 @@ context('Create a licence', () => {
         .nextInput()
         .enterAddress()
         .clickContinue()
+        .checkIfChangeLinkVisible()
+        .checkIfDeleteLinkVisible()
```

Comment:
would be nicer if we could parameterise this with the selected condition.. so it's more obvious..

if you think about the ideal way to express this test:
```
   stubConditions(
        {code: "xxxx1", requiresInput: true},
        {code: "xxxx2", requiresInput: false},
    )
   ... 

    checkIfChangeLinkVisible("xxxx1")
    checkIfDeleteLinkVisible("xxxx2")
```